ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-bgp.md

```diff
@@ -1429,6 +1440,12 @@
             # IPv6 prefix "A:B:C:D:E:F:G:H/I"
           - prefix: <str; required; unique>
             route_map: <str>
+        redistribute_routes:
```

Comment:
same as above

Reply:
RCF is only available for "isis" here:

```
s1-spine1(config-router-bgp)#address-family ipv6 multicast 
s1-spine1(config-router-bgp-af)#redistribute isis rcf ?
  FUNCNAME()  Routing control function name
  ```
  I have added a condition in j2 file:
  https://github.com/aristanetworks/avd/pull/3821/files#diff-b389418c101262db3e5545c20387bfabb1eda7c9ad8a7a1bc21b212be2252b93R987
  
  May be we can add this in description, I will do that.